src/data/stig_multipliers.ts

```diff
@@ -980,6 +980,66 @@ const DPS_STIG_MULTIPLIERS = Object.freeze({
       }),
     ],
   },
+  Himeko: {
+    T: [
+      new Multiplier({
+        active: true,
+        name: "Himeko T",
+        type: Type.TdmDealt,
+        value: 20,
+      }),
+      new Multiplier({
+        active: true,
+        name: "Himeko T: Boost fire DMG in Herrscher form",
+        type: Type.EleDealt,
+        value: 50,
+      }),
+    ],
+    M: [
+      new Multiplier({
+        active: true,
+        name: "Himeko M: Enemies take bonus Fire DMG from host when host loses HP",
+        type: Type.EleTakenHost,
+        value: 20,
+      }),
+    ],
+    B: [
+      new Multiplier({
+        active: true,
+        name: "Himeko B",
+        type: Type.TdmDealt,
+        value: 10,
+      }),
+      new Multiplier({
+        active: true,
+        name: "Himeko B: Every 10% of HP lost, boost fire DMG per stack (4 max)",
+        type: Type.EleDealt,
+        value: 40,
+      }),
+    ],
+    "2 set": [
+      new Multiplier({
+        active: true,
+        name: "Himeko 2 set: B-piece stacks give another 15% fire DMG",
+        type: Type.EleDealt,
+        value: 60,
```

Comment:
I think we'd either want to change this to 20 (the extra amount added), or keep it at 60, but set the B piece multiplier to off by default, up to you. Otherwise there's more than expected by default.

Reply:
Wait, I read it as +15 on top of the already existing +10 from B, for a total of +25 per stack; which in retrospect sounds insanely strong. I'll ask around in discord to get some other input on this.

Reply:
Yeah, second opinion says additional 15 per stack, so it's 60 total just from the 3pc bonus assuming 4 stacks. 100 total including B stacks

Reply:
Hot damn, you're right.